app/src/main/java/org/p2p/wallet/restore/ui/derivable/DerivableAccountsPresenter.kt

```diff
@@ -62,12 +62,7 @@ class DerivableAccountsPresenter(
                 val keys = secretKeys.map { it.text }
                 secretKeyInteractor.createAndSaveAccount(path, keys)
                 analytics.logWalletRestored(ScreenNames.OnBoarding.IMPORT_MANUAL)
-                val usernameExists = usernameInteractor.usernameExists()
```

Comment:
I would rather comment this logic with reference to the task, but ok

the target branch should be release